src/screens/ChannelListScreen/index.tsx

```diff
@@ -21,7 +20,6 @@ import {COLORS} from '../../utils/theme';
 const ChannelListScreen = ({route}) => {
   const {refresh} = useLocalDatabaseHook();
   const {checkNotificationPermission} = useRootChannelListHook();
-  const {handleFollow, isInitialFollowing} = useFollowUser();
   const {profile} = useUserAuthHook();
   const navigation = useNavigation();
   const isFocused = navigation.isFocused();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleFollow` and `isInitialFollowing` functions from the `useFollowUser` hook are no longer being used. Ensure that their functionalities are covered by other parts of the code.
```diff
-  const {handleFollow, isInitialFollowing} = useFollowUser();
```

47:
The `MessageChannelItem` component now only receives two props: `item` and `onChannelPressed`. The `hasFollowButton` and `handleFollow` props have been removed. This change simplifies the component's interface, but make sure that the follow functionality is still correctly implemented.
```diff
- return <MessageChannelItem item={item} onChannelPressed={() => goToChatScreen(item)} />;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->